website/src/components/visualisation/Resources/LineGraph.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { Brush, ResponsiveContainer, LineChart, Line, YAxis, XAxis, CartesianGrid, Tooltip, Legend, Surface, Symbols, ReferenceLine } from 'recharts';
+import outputJSON from '../../../output/output.json';
+import _ from "lodash";
+
+
+class LineRechartComponent extends React.Component<{}, any> {
+
+    //data = outputJSON;
+
+    constructor(props){
+        super(props);
+        this.state = {
+            disabled: [],
+            lineColours: {
+                team1: "#0095FF", 
```

Comment:
#234 merged, so we don't need to hardcode these now. for colours it's a bit of a pain but just put in some random colours after?

Reply:
(I'd personally write soemthing that generates colours based on a spectrum, but that's too much work. you can just make it loop around 6 colours as well if it's labelled ok.